testing/test_assertion.py

```diff
@@ -137,12 +137,12 @@ def test_foo(pytestconfig):
     def test_pytest_plugins_rewrite_module_names_correctly(self, testdir):
         """Test that we match files correctly when they are marked for rewriting (#2939)."""
         contents = {
-            "conftest.py": """
+            "conftest.py": """\
                 pytest_plugins = "ham"
             """,
             "ham.py": "",
             "hamster.py": "",
-            "test_foo.py": """
+            "test_foo.py": """\
```

Comment:
Was the escaped newline required for some reason?

Reply:
no but it makes the line numbers add up and makes my editor less angry about tqs

Reply:
tqs?

Reply:
triple quoted strings